### ISSUE TITLE
Documents how new tasking projects are setup

### ISSUE THREAD

**Issue by thomas-maschler** [crowdcover/logging-roads#13]
We will need to know how to set up new tasking projects (for once the validation process is over)

**Comment by jameslaneconkling** [comment]
yes.  I think we have all projects up for **Phase I** on tasks.hothosm.org.  Yes?

Adding new **Phase II** projects to tasks.loggingroads.org is pretty simple.  However, linking to those projects requires some minor modifications to the jekyll site as well (enable the 'start mapping' link for that country)--really simple to do, but you have to run Jekyll locally.  I can document Jekyll setup and editing, but might be easier for me to add them when needed--wouldn't take more than 2 min to make the change.

I'm sure Rafael is willing to help as well.
